Gateways/Mollie/MollieGateway.php

```diff
@@ -151,12 +165,20 @@ public function createOrder(array $requestData)
         try {
             return $this->apiClient->orders->create($requestData);
         } catch (ApiException $ex) {
+            $anonymizedRequest = $this->mollieOrderAnonymizer->anonymize($requestData);
+            $this->logger->critical($ex->getMessage(), $anonymizedRequest);
```

Comment:
i dont think this should be logged in here?
wasnt it before that the exception is passed on and then logged in the controller/action scope?
then we have double-logs here with the new implementation?

Reply:
ok found it
so the concept is full logs on the outer scope
inner-scope is detailed logs in special use cases (like the invalid total amount that was done before).

problem, as seen on the screenshot, you've opened a new catch way in the controller action that doesnt contain a log. if one would remove inner logs, we have a way that is not logged at all anymore.

please add the log to the outer scope.
if it has to stick also in this gateway for the anonymous data, then we would indeed need 2 logs, and we could turn this one to DEBUG maybe? because it shows in-depth additional data, and make the outer-one the error one?

<img width="913" alt="Screen Shot 2022-09-14 at 12 46 41" src="https://user-images.githubusercontent.com/5579326/190134315-261b118e-fe9d-4dec-b664-a321536b577a.png">

Reply:
Ineed i had to log in inner scope because the data was availiable there, so i made a debug message in inner scope and error on outerscope without details